### PR TITLE
Typo in head-jsonld.tpl, shop logo link : correction for #23151

### PR DESCRIPTION
**Pull request by fdonnet**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop 
| Description?      | Typo detected in #23151 (merged) by mgielecinski , thx to him
| Type?             | bug fix 
| Category?         | FO 
| BC breaks?        |  no
| Deprecations?     |  no
| Fixed ticket?     | Fixes #23151
| How to test?      | Logo url was not good in JSON ld declaration for index page
| Possible impacts? | Error

Simple typo correction.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24233)

### CODE REVIEW

File: themes/classic/templates/_partials/microdata/head-jsonld.tpl [hunk 1]
```diff
@@ -57,7 +57,7 @@
       "url" : "{$urls.pages.index}",
       "image": {
         "@type": "ImageObject",
-        "url":"{$urls.shop_domain_url}{$shop.logo}"
+        "url":"{$shop.logo}"
```
Comment (NeOMakinG) [inline themes/classic/templates/_partials/microdata/head-jsonld.tpl:60]: Damn, didn't see this one 👍

**Review by NeOMakinG — DISMISSED** [review]

**Comment by NeOMakinG** [comment]
@fdonnet, please target 1.7.8.x, overwise the error will be on the 1.7.8 version :/

**Review by PululuK — DISMISSED** [review]

File: themes/classic/templates/_partials/microdata/head-jsonld.tpl (hunk 1 above)
Comment (fdonnet) [reply themes/classic/templates/_partials/microdata/head-jsonld.tpl:60]: mgielecinski detected it :)

**Comment by fdonnet** [comment]
> 
> 
> @fdonnet, please target 1.7.8.x, overwise the error will be on the 1.7.8 version :/

@NeOMakinG how I can do it ? Can someone changes the target for my commit for me or need I remake a pull request on the correct branch ?

**Comment by NeOMakinG** [comment]
> > @fdonnet, please target 1.7.8.x, overwise the error will be on the 1.7.8 version :/
> 
> @NeOMakinG how I can do it ? Can someone changes the target for my commit for me or need I remake a pull request on the correct branch ?

This is the perfect time to learn it!

You need to get the 1.7.8.x branch locally

If you don't have a remote called upstream linked to the main repository :

`git remote add upstream git@github.com:PrestaShop/PrestaShop.git`

Then you need to fetch every upstream branches :

`git fetch upstream`

After it, just checkout 1.7.8.x : `git checkout --track upstream/1.7.8.x`

Then checkout your branch : `git checkout newfeature`

Then rebase it into 1.7.8.x: `git rebase 1.7.8.x`

If it contains conflict, resolve it by editing the file and keeping what you really want by using `git add`, then `git rebase --continue` until you've no conflict left

then force push it to your branches : `git push origin newfeature -f`

And then change the target of this PR into 1.7.8.x

You're done 🥳

**Comment by fdonnet** [comment]
@NeOMakinG, thx a lot,  I think I did something strange in my conflict resolution in the file tests/Unit/Classes/ToolsTest.php

**Comment by NeOMakinG** [comment]
Well, something went wrong about your rebase 😄 

Just open another PR, overwise I'll rebase this one later

**Comment by fdonnet** [comment]
@NeOMakinG new one is #23151 : target 1.7.8.x (sry for the garbage rebase)

**Pull request closed without merging**